examples/services_examples/find_chain_tip_header/find_chain_tip_header.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log/slog"
+
+	"github.com/bsv-blockchain/go-wallet-toolbox/examples/internal/show"
+	"github.com/bsv-blockchain/go-wallet-toolbox/pkg/defs"
+	"github.com/bsv-blockchain/go-wallet-toolbox/pkg/services"
+)
+
+func main() {
+	show.ProcessStart("Find Chain Tip Header")
+
+	cfg := defs.DefaultServicesConfig(defs.NetworkMainnet)
+	cfg.BHS.URL = "http://localhost:8080/api/v1"
```

Comment:
```suggestion
	cfg.BHS.URL = "http://localhost:8080"
```
`Block Headers Service API (URL: http://localhost:8080/api/v1/api/v1/chain/tip/longest): status code 404`